packages/iov-tendermint-rpc/src/v0-20/hasher.ts

```diff
@@ -1,22 +1,22 @@
-// import Buffer from "buffer";
-import hasher from "ripemd160";
-
+import { Ripemd160 } from "@iov/crypto";
 import { PostableBytes, TxId } from "@iov/tendermint-types";
 
-// a bit ugly as this expects buffer, while the rest of code uint8array
-export const hashTx = (tx: PostableBytes): TxId => {
-  const hash = new hasher().update(Buffer.from(prefix(tx))).update(Buffer.from(tx));
-  return Uint8Array.from(hash.digest()) as TxId;
-};
+export function hashTx(tx: PostableBytes): TxId {
+  const hash = new Ripemd160()
```

Comment:
personal preference....

maybe pass prefix(tx) in the constructor and this fits on one line?

Reply:
Jap, works and saves 3 lines, nice